aries_cloudagent/messaging/jsonld/routes.py

```diff
@@ -37,6 +37,7 @@ class SignRequestSchema(OpenAPISchema):
                             "creator": fields.Str(required=False),
                             "verificationMethod": fields.Str(required=False),
                             "proofPurpose": fields.Str(required=False),
+                            "type": fields.Str(required=False),
```

Comment:
I ended up looking at this issue as well before seeing you'd gotten here first but in my fix I made `verificationMethod` and `proofPurpose` required. Based on my reading of the VC data model and LD Proof spec, these are required values that, in the context of these routes at least, we don't know how to populate automatically. Shouldn't we make these attributes required?

Reply:
Yes, good point. I'll update the PR later today.